keras2onnx/main.py

```diff
@@ -75,6 +75,14 @@ def convert_keras(model, name=None, doc_string='', target_opset=None, channel_fi
     set_logger_level(logging.DEBUG if debug_mode else logging.INFO)
     tf2onnx.logging.set_level(logging.DEBUG if debug_mode else logging.INFO)
 
+    if isinstance(model, tf.keras.Model):
+        if not is_tf_keras:
+            k2o_logger().warning("WARNING: This is a tensorflow keras model, but keras standalone converter is used." +
+                                 " Please set environment variable TF_KERAS.")
```

Comment:
Specify TF_KERAS=1

Reply:
As long as the environment variable is set, then it will use tf.keras.